tests/test_model_wrapper.py

```diff
@@ -77,43 +92,63 @@ def test_wrap_scikit_keras_regression_model(self, housing):
         train_regression_model_pandas(create_scikit_keras_regressor, housing)
 
 
-def train_classification_model_numpy(model_initializer, dataset):
+def train_classification_model_numpy(model_initializer, dataset,
+                                     use_dataset_wrapper=True):
```

Comment:
for my understanding why is wrapper here not optional?

Reply:
Yes, we are making the DatasetWrapper optional with this PR, so the user doesn't have to wrap their dataset in a DatasetWrapper when they call the wrap_model function.